src/Tizen.NUI.Components/Controls/Scrollbar.cs

```diff
@@ -184,7 +184,65 @@ public enum DirectionType
         /// Get style of scrollbar.
         /// </summary>
         /// <since_tizen> 8 </since_tizen>
-        public new ScrollBarStyle Style => ViewStyle as ScrollBarStyle;
+        public new ScrollBarStyle Style
+        {
+            get
+            {
+                return new ScrollBarStyle(ViewStyle as ScrollBarStyle);
+            }
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ImageView TrackImage
```

Comment:
Hello,
please add comments to explain what is TrackImage

Reply:
OK I will add the comments